### PR TITLE
modulized_footer_and_general_improvments

### PR DESCRIPTION
**Pull request by adamseth2**
- removed boiler plate for moduleHtml (don't need it)
- redesigned and modulized footer
- changed navbar's tag to nav instead of section

TODO:
- make the year update automatically

### CODE REVIEW

File: about_us.html [hunk 1]
```diff
@@ -247,137 +265,110 @@ <h1>
...
+                        <img
+                            src="./assets/images/unslanted_outreach_2016.jpg"
+                        />
                     </div>
                 </div>
             </div>
 
             <div class="space-after-rows"></div>
         </section>
+        <h1></h1>
```
Comment (VladimirKupryukhin) [inline about_us.html]: Whats the point of `<h1></h1>`? I don't see this used anywhere else like that

File: assets/scss/default.scss [hunk 2]
```diff
@@ -24,6 +24,65 @@ nav {
...
+    &::after {
+        content: '';
+        position: absolute;
+        top: -50px;
+        left: 0;
+        height: fit-content;
+        height: 100px;
+        width: 100%;
+
+        background-image: url('../images/line_divider_svg.svg');
```
Comment (VladimirKupryukhin) [inline assets/scss/default.scss:42]: You should figure out how to make this divider svg resize as the view window becomes smaller. It looks somewhat awkward to have a large/tall divider on a phone.

**Comment by VladimirKupryukhin** [comment]
You forgot to change the footer for the "Projects" page.

File: about_us.html (hunk 1 above)
Comment (adamseth2) [reply about_us.html]: removed

File: assets/scss/default.scss (hunk 2 above)
Comment (adamseth2) [reply assets/scss/default.scss:42]: I made it responsive

**Comment by adamseth2** [comment]
- Fixed issues above
- made copyright year dynamic using JS
- increase gap for dividing line for footer and reduce size for mobile

**Review by VladimirKupryukhin — APPROVED** [review]
Everything looks good now. You can merge

**Pull request merged**